torch/onnx/symbolic_opset12.py

```diff
@@ -9,7 +10,56 @@
 
 # This file exports ONNX ops for opset 12
 
+black_listed_operators = [
+    "ArgMin", "ArgMax"
+]
+
 @parse_args('s', 'v')
 def einsum(g, equation, tensor_list):
     tensors = sym_help._unpack_list(tensor_list)
     return g.op("Einsum", *tensors, equation_s=equation)
+
+@parse_args('v', 'f', 'i')
+def dropout(g, input, p, train):
+    # in eval mode, dropout is non-op - if the node's train param is set to False, dropout is non-op
+    if not sym_help._training_mode:
+        return input
+    p = g.op("Constant", value_t=torch.tensor(p))
+    return g.op("Dropout", input, p, outputs=1)
+
+
+@parse_args('v', 'v', 'v', 'v', 'v', 'i', 'f', 'f', 'i')
+def batch_norm(g, input, weight, bias, running_mean, running_var, training, momentum, eps, cudnn_enabled):
+    input_sizes = input.type().sizes()
+
+    if weight is None or sym_help._is_none(weight):
+        assert len(input_sizes) > 1
+        weight_value = torch.tensor([1.] * input_sizes[1]).type(
+            'torch.' + input.type().scalarType() + 'Tensor')
+        weight = g.op("Constant", value_t=weight_value)
```

Comment:
Is this a parameter that is expected to be trainable in the training-amenable mode? If yes, then if these are going to be translated to onnx::constant, then that would be a problem. We should try and keep all trainable parameters as initializers, because onnx::constant would mean that they are immutable.